modules/veteran_confirmation/VETERAN_CONFIRMATION.yml

```
@@ -15,12 +15,15 @@ info:
 
     ### 2. Test the API
     In the endpoint documentation below, we've provided a curl command builder for trying out the API before implementation with your app.
+    Use [Test User](https://github.com/department-of-veterans-affairs/vets-api-clients/blob/master/test_accounts.md) attributes to populate the curl.
```

```suggestion
    Use [Test User](https://github.com/department-of-veterans-affairs/vets-api-clients/blob/master/test_accounts.md) attributes to populate the request body.
```

I think we have been overloading `curl` in a lot of places (it's both the tool and the command line invocation itself etc.). Also, "Request Body" is the header that actually shows where these attributes actually go.